### DISCUSSION TITLE
更新導致使用者字庫消失的問題

### DISCUSSION THREAD

**Discussion by larryjclai** [vChewing/vChewing-macOS#148] [Q&A]
我最近從 2.70 升級到 2.75 的時候發現我原本的自定義使用者字庫消失了（使用 shift + Enter 加入字庫的字詞）
請問之後在升級的時候有什麼地方需要注意，才不會讓字庫要重新建立一次嗎？
我的升級方式是直接下載 DMG 檔案重新安裝

謝謝～

**Comment by ShikiSuen** [comment]
@larryjclai 威注音沒有 DMG 版本。
另外，請在偏好設定內手動重新設定使用者辭典目錄。
原廠預設的辭典目錄是：
~/Library/Containers/org.atelierInmu.inputmethod.vChewing/Data/Library/Application\ Support/vChewing/

**Reply by larryjclai** [reply]
抱歉，好像是 pkg 才對（就是直接下載然後重新安裝）
我看目錄的確是那個沒錯，我下次升級的時候再注意看看是否哪裡有搞錯了。
謝謝回覆～

**Reply by ShikiSuen** [reply]
XD 你現在就可以拿 2.8.0 版試試看：https://github.com/vChewing/vChewing-macOS/releases/tag/2.8.0

**Reply by ShikiSuen** [reply]
> 抱歉，好像是 pkg 才對（就是直接下載然後重新安裝） 我看目錄的確是那個沒錯，我下次升級的時候再注意看看是否哪裡有搞錯了。 謝謝回覆～

我懷疑可能是因為 2.3.0 - 2.7.0 版的威注音的簽證問題導致沙箱書籤失效了。
你看你辭典目錄還是那個目錄，你也請重新設定一遍。這個過程會重建沙箱書籤。
